tests/unit/test_workflow_files.py

```diff
@@ -321,8 +320,8 @@ def test_parse_reg__ok(
         pytest.param(
             ('non_exist', False),
             'non_exist',
-            None,
-            None,
+            '{cylc_run_dir}/non_exist',
+            "no flow.cylc or suite.rc in ",
```

Comment:
This was deliberate - there should not be an error if no flow file exists because "online" commands (`src=False`) don't need it
```suggestion
            None,
```

Reply:
Right, understood now in light of your previous comment!

Reply:
Restored.